.github/workflows/config/spelling_allowlist.txt

```diff
@@ -313,3 +317,4 @@ verifier
 vertices
 wavefunction
 workflow
+NumPy
```

Comment:
Isn't the spelling checker supposed to accept anything in CamelCase as a legal spelling?

Reply:
> Isn't the spelling checker supposed to accept anything in CamelCase as a legal spelling?

Not that I know of. *Perhaps* if "Num" and "Py" were valid words, but those are not valid words either, so I wouldn't expect it to allow "NumPy".

Reply:
OK. I vaguely recall tweaking a regular expression somewhere once upon a time ... but it could be long gone by now.